server_addon/houdini/client/ayon_houdini/plugins/create/create_hda.py

```diff
@@ -56,8 +59,18 @@ def create_instance_node(
                 raise CreatorError(
                     "cannot create hda from node {}".format(to_hda))
 
+            # Pick a unique type name for HDA product per folder path per project.
+            type_name = (
+                "{project_name}{folder_path}_{node_name}".format(
+                    project_name=get_current_project_name(),
+                    folder_path=folder_path.replace("/","_"),
```

Comment:
@iLLiCiTiT 
is that a good way to get folder path with underscores instead of `/` ?

Reply:
have you tried `get_current_folder_path()`?

Reply:
we already have `folder_path`, 
I was just wondering if there's a better way instead of `folder_path.replace("/","_")` ?

Reply:
>  if there's a better way instead of folder_path.replace("/","_") ?

I mean - that does seem to be the best approach. Seems fine to me.